src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -191,7 +191,8 @@
                     "weight": "The weighting of a test case determines the points a student receives for passing the test case. The number of points a test case awards is proportional to its weight and calculated as a fraction of the total exercise points.",
                     "bonusMultiplier": "This multiplies the points given for passing a test case without affecting the fractions of other test cases. Multipliers greater than 1 can result in student scores greater than 100%.",
                     "bonusPoints": "Bonus points added on top of the regular points for passing a test case. Setting bonus points can result in student scores greater than 100%.",
-                    "visibility": "Feedback and points for test cases marked as ‘After Due Date’ only become visible to students after the due date has passed. Tests with visibility ‘Never’ do not influence the student score."
+                    "visibility": "Feedback and points for test cases marked as ‘After Due Date’ only become visible to students after the due date has passed. Tests with visibility ‘Never’ do not influence the student score.",
+                    "type": "The type of the test case determine if the test case has been generated with the Structure Oracle (STRUCTURAL), or has been written by the editor manually (BEHAVIORAL). This is currently only supported for exercises in Java, the tests in all other languages are of type DEFAULT."
```

Comment:
> has been generated with the Structure Oracle

I don't think this is a good way to write this. Structural test cases can also be created manually by editing the test.json file. (Same for the german translation)